Assets/Scripts/Attacks/PlayerBasicAttack.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Assets.Scripts.Attacks
+{
+    public class PlayerBasicAttack : MonoBehaviour
+    {
+        private GameObject arrowPrefab;
+        private GameObject arrowSpawningPosition;
```

Comment:
Can be converted to a Vector3. We don't need the whole GameObject. In the Awake method, access the transform.position of the gameObject.

Reply:
done

Reply:
didn't work, got the initial transform position of the player and arrows are shooting from that spot and not from the current player position

Reply:
Oh it stored its initial value and never updated? Ok then